.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: Run Lua unit tests and static code analyzer
         run: poetry run poe lua-tests
 
+      - name: Release language container
+        run: ./release_language_container.sh
```

Comment:
Thinking about, if we maybe should pass the export path (.build_output/exported_container/)  as a parameter to release_language_container.sh. Then we could define it as an environment variable in the GitHub Workflow and use it here in the upload.

Reply:
Aren't we going to make the release procedure similar to the Transformers Extension shortly? I so, why bother?

Reply:
OK, will do this for the learning purpose.